server/src-lib/Hasura/GraphQL/RemoteServer.hs

```diff
@@ -181,28 +181,28 @@ instance J.FromJSON (FromIntrospection G.GType) where
         G.TypeNamed _ n -> G.TypeNamed (G.Nullability False) n
 
 instance J.FromJSON (FromIntrospection G.InputValueDefinition) where
-  parseJSON = J.withObject "InputValueDefinition" $ \o -> do
+  parseJSON = J.withObject "RemoteSchemaInputValueDefinition" $ \o -> do
```

Comment:
What are the consequences of changing that name? Is there a risk it might break an existing stored JSON document somewhere?

Reply:
Oh, this change shouldn't have been there. The original one is fine. I will make the change 😓